### DISCUSSION TITLE
Running wezterm-mux-server on an old linux box

### DISCUSSION THREAD

**Discussion by j-xella** [wezterm/wezterm#3978] [Q&A] [answered]
I have a rather old headless Linux server (RHEL 7.9). I connect to it from Windows PC, where wezterm runs fine. All I really need is to be able to run wezterm-mux-server on the Linux box. 

However, I am unable to do so, whatever I try, I get something like:

```
WezTerm-20230408-112425-69ae8472-Ubuntu20.04.AppImage: /lib64/libm.so.6: version `GLIBC_2.27' not found (required by WezTerm-20230408-112425-69ae8472-Ubuntu20.04.AppImage)
```

I understand that the main terminal may require a lot of dependencies and hence it is not feasible to support old machines. But wezterm-mux-server should be relatively dependency free, no? maybe it would be possible to have a separate release artefact, just with wezterm-mux-server, but with very minimal requirements? musl target?

Also, shouldn't appimage be able to run almost anywhere? Maybe adding glibc library to it would do the trick?

Thank you in advance.

**Comment by wez** [comment, marked as answer]
AppImage's are built on ubuntu 20 and will run on linux distros that are essentially from the same era of glibc.  They are not fully containerized packages.

You should install the centos7 rpm for such an old system; the only reason I still build and provide those is literally for this sort of situation (https://github.com/wez/wezterm/issues/1947).

**Comment by j-xella** [comment]
Thanks, that worked!
